sdk/README.md

```diff
@@ -96,7 +98,17 @@ As of today, the below pipelineRun features are available within `dsl-compile-te
 To compile Kubeflow Pipelines as Tekton pipelineRun, simply add the `--generate-pipelinerun` as part of your `dsl-compile-tekton`commands. e.g.
 - `dsl-compile-tekton --py sdk/python/tests/compiler/testdata/tolerations.py --output pipeline.yaml --generate-pipelinerun`
 
+### 2. Compile Kubeflow Pipelines with artifact enabled
+
+Prerequisite: Install [Kubeflow Pipeline](https://www.kubeflow.org/docs/pipelines/installation/).
+
+By default, artifacts are disabled because it's depended on Kubeflow Pipeline's minio setup. When artifacts are enabled, all the output parameters are also treated as artifacts and persist to the default object storage. Enabling artifacts also allow files to be downloaded or stored as artifact inputs/outputs. Since artifacts are depending on the Kubeflow Pipeline's setup by default, the generated Tekton pipeline must be deployed to the same namespace as Kubeflow Pipeline.
+
+To compile Kubeflow Pipelines as Tekton pipelineRun, simply add the `--enable-artifacts` as part of your `dsl-compile-tekton` commands. e.g.
+- `dsl-compile-tekton --py sdk/python/tests/compiler/testdata/artifact_location.py --output pipeline.yaml --enable-artifacts`
+
```

Comment:
can we add one more line here to show how this must be run and what the output should look like:
- `$ kubectl apply -f pipeline.yaml`
- `$ tkn pipeline start custom-artifact-location-pipeline --showlog -n kubeflow`
    ```
    ? Value for param `secret_name` of type `string`? (Default is `mlpipeline-minio-artifact`) mlpipeline-minio-artifact
    ? Value for param `tag` of type `string`? (Default is `1.31.0`) 1.31.0
    ? Value for param `namespace` of type `string`? (Default is `kubeflow`) kubeflow
    ? Value for param `bucket` of type `string`? (Default is `mlpipeline`) mlpipeline
    Pipelinerun started: custom-artifact-location-pipeline-run-qtxd5
    Waiting for logs to be available...
    
    [foo : copy-artifacts] Added `storage` successfully.
    [foo : copy-artifacts] `/tekton/results/output` -> `storage/mlpipeline/runs/custom-artifact-location-pipeline-run-qtxd5/custom-artifact-location-pipeline-run-qtxd5-foo-rlx2z-pod-tl7nn/output.txt`
    [foo : copy-artifacts] Total: 0 B, Transferred: 6 B, Speed: 554 B/s
    ```
but then we should rename `foo` into smth meaningful, see other comment